src/door.rs

```diff
@@ -0,0 +1,291 @@
+                    std::process::exit(1);
+                }
+            };
+
+            match templates.write_dos("doorman.bat", &node_rundir, &commands) {
+                Ok(_) => (),
+                Err(e) => {
+                    eprintln!("{0}", e);
+                    std::process::exit(1);
+                }
+            };
+
+            let term = match env::var_os("TERM") {
+                Some(val) => match val.into_string() {
+                    Ok(val) => val,
+                    Err(_) => String::from("xterm"),
+                },
+                None => String::from("xterm"),
+            };
+
+            let run = match Command::new("docker")
+                .arg("run")
+                .arg("-d")
+                .arg("-p5901:5901")
+                .arg(format!("-v{0}:/mnt/doorman", node_rundir.display()))
+                .arg(format!("-v{0}:/mnt/door", door.path))
+                .arg(format!("-eTERM={0}", term))
+                .arg(format!("-eDOORMAN_DOOR={0}", args.door))
+                .arg(format!("-eDOORMAN_USER={0}", user.username))
+                .arg(format!("-eDOORMAN_UID={0}", user.uid))
+                .arg(format!(
+                    "-eDOORMAN_SYSOP={0}",
+                    if user.is_sysop { "1" } else { "0" }
+                ))
+                .arg(format!(
+                    "-eDOORMAN_RAW={0}",
+                    if args.raw { "1" } else { "0" }
+                ))
+                .arg(config.doorman.dosemu_container.as_str())
+                .arg("wait-for-launch.sh")
+                .stdout(Stdio::piped())
+                .spawn()
+            {
+                Ok(run) => run,
+                Err(e) => {
+                    eprintln!("Couldn't start container: {0}", e);
+                    std::process::exit(1);
+                }
+            };
+
+            let container_id = get_container_id(run);
+            println!("Container ID = {0}", container_id.trim());
+
+            match Command::new("docker")
+                .arg("exec")
+                .arg("-ti")
+                .arg(container_id.trim())
+                .arg("launch.sh")
+                .status()
+            {
+                Ok(status) => match status.code() {
+                    Some(code) => std::process::exit(code),
+                    None => {
+                        eprintln!("No exit status from client");
+                        std::process::exit(1);
+                    }
+                },
+                Err(e) => {
+                    eprintln!("Couldn't start client: {0}", e);
+                    std::process::exit(1);
+                }
+            };
+        } else {
+            node += 1;
+        }
+    }
+
+    if !found_node {
+        eprintln!("All nodes for {0} are busy!", args.door);
+        std::process::exit(1);
+    }
+}
+
+pub fn configure(args: &SysopCmdArgs, config: &Config) {
+    println!("LOL not implemented yet");
+    return;
+}
```

Comment:
```suggestion
pub fn configure(args: &SysopCmdArgs, config: &Config) {
    unimplemented!()
}
```
You can use the `unimplemented!()` or the `todo!()` macro to indicate that the implementation is missing for now. Note that this will panic when executing.